autobot-backend/api/user_management/password_change_test.py

```
@@ -0,0 +1,511 @@
+        mock_rate_limiter,
+    ):
+        """THE takeover case: an authenticated non-admin, non-owner caller
+        posts only ``new_password`` at another user's id. This must be
+        rejected before the service is ever called -- the vulnerable code
+        would have accepted this and silently disabled verification."""
+        attacker_id = uuid.uuid4()
+        current_user = {"user_id": str(attacker_id), "token": "attacker.jwt"}
+        context = TenantContext(user_id=attacker_id, is_platform_admin=False)
+
+        with _patched_limiter(mock_rate_limiter):
+            pass
```

Ruff (0.16.3)

[warning] Unnecessary `pass` statement

Remove unnecessary `pass`

(PIE790)